src/controller/NotificationController.js

```diff
@@ -0,0 +1,116 @@
+import dotenv from 'dotenv';
+import NotificationUtil from '../utilities/Notification';
+
+dotenv.config();
+
+
+/**
+ * @description class for controlling notifications
+ */
```

Comment:
I feel a good description will be "@description class definition for handling notifications". I think its clearer and easier to understand than the one you wrote on line 8.

Reply:
Noted. I have changed it accordingly.